trello-solarized-dark.user.css

```diff
@@ -1,3 +1,14 @@
+/* ==UserStyle==
+@name         Trello Solarized Dark Grey
+@version      0.6.0
+@description  A Dark Travis-CI style with purple highlights
```

Comment:
Change the description to "Trello board dark user style inspired by Ethan Schoonover's Solarized Dark color palette."

Reply:
Oops! Copy paste error from another user style :) Added a commit to fix it.